src/errorcodes.xml

```diff
@@ -3399,7 +3399,7 @@
       <message>
         An attempt was made to initialize a new transaction within the scope of an active transaction.
         Creating such nested transactions is not supported.
-        Consider reusing the already active transaction or running the new transaction outside the already active transaction's scope.
```

Comment:
Not really a typo, just means a slightly different thing. "the new" refers to the new transaction as introduced in the first sentence. "a new" would mean a (possibly) different transaction. Keep it like this if you want, however. Doesn't make that much difference.